### ISSUE TITLE
Access to source materials?

### ISSUE THREAD

**Issue by SeyelentEco** [dohliam/hawaiian-corpus#1]
I'm working on a project that requires similar data in order to build models to aim with improving readability of older Hawaiian language texts, do you still have access to the source data?  I am working on building the dataset myself but I stumbled upon your data and I thought it might save me some steps.

**Comment by bee-san** [comment]
Hi @SeyelentEco , I cannot help you with source data but to necro this...

Did you build anything for older hawaiian texts? I am also very interested in this and curious as to what you did :)

**Comment by SeyelentEco** [comment]
Yes I used existing text from Wehewehe and removed 'okina and kahakos from existing text and then fine-tuned a ChatGPT4 model to learn how to add them back.  You can play with it here: https://olelotools.com/
